### PR TITLE
Update gnsi-acctz.yang

### PR DESCRIPTION
**Pull request by marcushines**
Remove duplicate container for counters

### CODE REVIEW

File: acctz/gnsi-acctz.yang [hunk 1]
```diff
@@ -160,21 +160,8 @@ module gnsi-acctz {
 
     grouping grpc-server-acctz-counters {
         description
-          "A collection of counters from the gNSI.acctz module.";
-
-        container counters {
```
Comment (greg-dennis) [inline acctz/gnsi-acctz.yang:165]: This leaves one too many closing braces in place.

File: acctz/gnsi-acctz.yang [hunk 2]
```diff
@@ -160,21 +160,8 @@ module gnsi-acctz {
...
-                type oc-types:timeticks64;
-                description
-                    "The last time that the counters were cleared (reset to
-                     zero). This value is reported as nanoseconds since epoch
-                     (January 1st, 1970 00:00:00 GMT).";
-            }
+          "A collection of counters from the gNSI.acctz module. This collection
+           augments the existing certz grpc counters";
 
             container client-counters {
```
Comment (greg-dennis) [inline acctz/gnsi-acctz.yang:166]: un-indent these children of the counters container?

File: acctz/gnsi-acctz.yang [hunk 3]
```diff
@@ -149,39 +155,25 @@ module gnsi-acctz {
...
+          "A collection of counters from the gNSI.acctz module. This collection
+           augments the existing certz grpc counters";
 
-        container counters {
-            description
-              "A collection of counters from the gNSI.acctz module
-               for acctz clients and sources.";
-           config false;
-
-            leaf last-cleared-on {
```
Comment (greg-dennis) [inline acctz/gnsi-acctz.yang:171]: It's ok for this leaf to no longer exist anywhere?

File: acctz/gnsi-acctz.yang [hunk 4]
```diff
@@ -25,6 +25,12 @@ module gnsi-acctz {
         "This module provides a data model for the metadata of the gRPC
          accounting operations on a device.";
 
+    revision 2023-01-24 {
+        description
+            "Remove duplicate counter container.";
```
Comment (greg-dennis) [inline acctz/gnsi-acctz.yang:30]: "counter" -> "counters"

**Review by greg-dennis — CHANGES_REQUESTED** [review]

**Comment by wenovus** [comment]
We're looking at resolving these issues at https://github.com/openconfig/public/pull/1037

Propose to close this PR and move comments over there.

**Pull request closed without merging**